index.html

```diff
@@ -0,0 +1,153 @@
+                <li class="s-list"><a href="#">English</a></li>
+                <li class="s-list"><a href="#">My Page</a></li>
+                <li class="s-list"><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <header>
+            <nav class="container">
+                <div class="logoDiv">
+                    <a class="logo font-face" href="index.html">&lt;/&gt;CodeWorld</a>
+                </div>
+
+                <img class="mobile-menu pb-4" id="mobileMenu" src="images/ic_menu.svg" alt="Open Navigation">
+                
+                <div class="nav-list">
+                    <span id="mobile-exit">&times;</span>
+                    <ul>
+                        <li class="p-cta"><a href="about.html">About</a></li>
+                        <li class="p-cta"><a href="#">Program</a></li>
+                        <li class="p-cta"><a href="#">join</a></li>
+                        <li class="p-cta"><a href="#">Sponsor</a></li>
+                        <li class="p-cta"><a href="#">News</a></li>
+                        <li class="p-cta call2action"><a href="#">Start Now</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <div class="container">
+            <div class="headline">
+                <p class="intro font-face">Welcome to "CodeWorld"</p>
+                <h1 class="title font-face">WEB DEVELOPMENT TO BOOST YOUR CARRER</h1>
+                <p class="discription font-face">We prove you with all the tool that will enable you to grow in the web development field. We konw that web development field grows everyday with our course and community you will manage to catch up and stay up to speed.</p>
+                <p class="date font-face">COURSE DURATION IS 10 MONTHS + LIFE TIME SUPPORT</p>
+                <p class="location font-face">We are fully online | Support team available 24 hours |</p>
+            </div>
+        </div>
+    </section>
+
+    <section class="main-program">
+        <div class="program-bg"></div>
+        <div class="p-cont">
+            <p class="program-title">Main Program</p>
+            <hr>
+        </div>
+        
+        <div class="container grid">
+            <div class="card g-col-12 g-col-md-2">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/lecturer.png" alt="lecture image">
+                    <p class="t1">Lecture</p>
+                </div>
+                <p class="program-disc">Watch some of our meterials from various expect across the world.</p>
+            </div>
+            <div class="card g-col-12 g-col-md-2">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/board.png" alt="lecture image">
+                    <p class="t1">CC Exhibition</p>
+                </div>
+                <p class="program-disc">Our board contain timetables and announcements.</p>
+            </div>
```

Comment:
-  [ ] [REQUIRED] Kindly note that the cards need not be separated by little spaces each, just as it is on the behance design.
### Your Implimentation
![Screenshot from 2022-04-21 17-21-56](https://user-images.githubusercontent.com/58692590/164507092-9073b60e-7c55-46f0-8521-88a1e4d4aab5.png)

### Behance Design
![Screenshot from 2022-04-21 17-21-55](https://user-images.githubusercontent.com/58692590/164507037-169d58f0-6e5c-4e5d-bef6-a052e058d2ff.png)